bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleEngine.java

```diff
@@ -916,10 +941,6 @@ private void updateContext(String ruleUID, String moduleUID, Map<String, ?> outp
     /**
      * @return copy of current context in rule engine
      */
-    private Map<String, Object> getContext(String ruleUID) {
```

Comment:
Why did you remove it? In half of all calls to `getContext` the connections are `null`, so this method seemed to be worthwhile.

Reply:
To simplify and optimize the code; "half of all calls" is just 2, but if you think this is important, I can revert it